pkg/odo/cli/component/create.go

```diff
@@ -480,10 +482,10 @@ func (co *CreateOptions) Complete(name string, cmd *cobra.Command, args []string
 		co.devfileMetadata.componentName = strings.ToLower(componentName)
 		co.devfileMetadata.componentNamespace = strings.ToLower(componentNamespace)
 
-		// Categorize the sections
-		log.Info("Validation")
-
 		if util.CheckPathExists(DevfilePath) || co.devfileMetadata.devfilePath.value != "" {
+			// Categorize the sections
+			log.Info("Validation")
```

Comment:
Why we add this message in each code block? If the two blocks are mutual exclusive then we should add the message globally, I test both direct mode and interactive mode it works fine for me.

Reply:
discussed with Jingfu, more info here https://github.com/openshift/odo/pull/3291#discussion_r442324400